rsmtool/notebooks/data_description.ipynb

```diff
@@ -7,6 +7,20 @@
     "## Description of the data"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# print the excluded features in the report if exists.\n",
+    "if excluded_features:\n",
+    "    if len(excluded_features) == 1:\n",
+    "        display(Markdown(f\"The following feature was excluded because its standard devision in the training set is equal to 0: **{excluded_features[0]}**.\"))\n",
+    "    else:\n",
+    "        display(Markdown(f\"The following features were excluded because their standard devision in the training set is equal to 0: **{\", \".join(excluded_features)}**.\"))"
+   ]
```

Comment:
I don't think you need an if statement.  Just say: `The following feature(s) were excluded because their standard devision in the training set is equal to 0:`. That should work in either case. 

Also, I am getting an error when I run this – something about a missing `}`.

Reply:
hmm. Interesting. May I ask how did you run it? I didn't see a missing }.